pkg/sdk/model/output/splunk_hec.go

```diff
@@ -0,0 +1,168 @@
+	// Identifier for the Splunk index to be used for indexing events. If this parameter is not set, the indexer is chosen by HEC. Cannot set both index and index_key parameters at the same time.
+	Index string `json:"index,omitempty"`
+	// The field name that contains the Splunk index name. Cannot set both index and index_key parameters at the same time.
+	IndexKey string `json:"index_key,omitempty"`
+	// The host location for events. Cannot set both host and host_key parameters at the same time. (Default:hostname)
+	Host string `json:"host,omitempty"`
+	// Key for the host location. Cannot set both host and host_key parameters at the same time.
+	HostKey string `json:"host_key,omitempty"`
+	// The source field for events. If this parameter is not set, the source will be decided by HEC. Cannot set both source and source_key parameters at the same time.
+	Source string `json:"source,omitempty"`
+	// Field name to contain source. Cannot set both source and source_key parameters at the same time.
+	SourceKey string `json:"source_key,omitempty"`
+	// The sourcetype field for events. When not set, the sourcetype is decided by HEC. Cannot set both source and source_key parameters at the same time.
+	SourceType string `json:"sourcetype,omitempty"`
+	// Field name that contains the sourcetype. Cannot set both source and source_key parameters at the same time.
+	SourceTypeKey string `json:"sourcetype_key,omitempty"`
+	// By default, all the fields used by the *_key parameters are removed from the original input events. To change this behavior, set this parameter to true. This parameter is set to false by default. When set to true, all fields defined in index_key, host_key, source_key, sourcetype_key, metric_name_key, and metric_value_key are saved in the original event.
+	KeepKeys bool `json:"keep_keys,omitempty"`
+	//If a connection has not been used for this number of seconds it will automatically be reset upon the next use to avoid attempting to send to a closed connection. nil means no timeout.
+	IdleTimeout int `json:"idle_timeout,omitempty"`
+	// The amount of time allowed between reading two chunks from the socket.
+	ReadTimeout int `json:"read_timeout,omitempty"`
+	// The amount of time to wait for a connection to be opened.
+	OpenTimeout int `json:"open_timeout,omitempty"`
+	// The path to a file containing a PEM-format CA certificate for this client.
+	ClientCert string `json:"client_cert,omitempty"`
+	// The private key for this client.'
+	ClientKey string `json:"client_key,omitempty"`
+	// The path to a file containing a PEM-format CA certificate.
+	CAFile string `json:"ca_file,omitempty"`
+	// The path to a directory containing CA certificates in PEM format.
+	CAPath string `json:"ca_path,omitempty"`
+	// List of SSL ciphers allowed.
+	SSLCiphers string `json:"ssl_ciphers,omitempty"`
+	// Indicates if insecure SSL connection is allowed (default:true)
+	InsecureSSL bool `json:"insecure_ssl,omitempty"`
```

Comment:
```suggestion
	InsecureSSL *bool `json:"insecure_ssl,omitempty"`
```
is this really true by default?

Reply:
Ah sorry, no its flase :) my bad